src/deps.ts

```diff
@@ -1,2 +1,2 @@
-export { readJson } from "https://deno.land/std@v0.52.0/fs/mod.ts";
-export { extname } from "https://deno.land/std@v0.52.0/path/mod.ts";
+export { readJson } from "https://deno.land/std@v0.53.0/fs/mod.ts";
```

Comment:
I'm sorry but I prefer to upgrade std version in a separate PR, it's easier to revert in case of bug introduced with the `std@v0.53.0` without revert the logs part.

Reply:
I removed